skema/program_analysis/CAST/python/node_helper.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Dict
+from skema.program_analysis.CAST2FN.model.cast import SourceRef
+
+from tree_sitter import Node
+
+CONTROL_CHARACTERS = [
```

Comment:
Python will likely have a distinct set of control characters. It is also possible that the control characters don't show up in the parse tree like they do for tree-sitter-fortran. It doesn't look you are using the control character helper functions at this point, but it may be something to be aware of in the future.

Reply:
Yeah, I just copied over what you had. I have plans to clean out the control characters that don't exist in Python and tailoring the list to Python's control structures. Thank you for the heads up!